src/azure/cli/commands/arm.py

```diff
@@ -394,12 +406,25 @@ def _update_instance(instance, part, path):
     try:
         index = index_regex.match(part)
```

Comment:
Not sure this is any less confusing, but I renamed it to "index_or_filter_regex"

---
In reply to: [76667898](https://github.com/Azure/azure-cli/pull/740#discussion_r76667898) [](ancestors = 76667898)